libs/algorithms/tests/unit/container_algorithms/foreach_adapt.cpp

```diff
@@ -0,0 +1,23 @@
+#include <hpx/hpx_main.hpp>
```

Comment:
Please add copyright, license, and SPDX tag.

Reply:
Sorry, missed it. Just fixed it and clang-formatted the code.